types/ember__runloop/ember__runloop-tests.ts

```diff
@@ -15,9 +15,13 @@ import {
     schedule,
     scheduleOnce,
     throttle,
+    Timer,
 } from "@ember/runloop";
 import { Backburner, DebugInfo, DeferredActionQueues, QueueItem } from "@ember/runloop/-private/backburner";
 
+declare const timer: Timer;
+timer; // $ExpectType Timer
```

Comment:
This doesn't seem to be testing anything anymore since we just declared the const as type `Timer`.

Reply:
Previously I used `$ExpectType string | number`, but after running `pnpm test ember__runloop` it gives this error:

```
error  TypeScript@4.6, 4.7, 4.8, 4.9, 5.0, 5.1, 5.2, 5.3, 5.4 expected type to be:
  string | number
got:
  Timer  @definitelytyped/expect
```

`Timer` is defined as `type Timer = string | number`

Any tips on how to fix?